invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_SEARCH_CLASS = "invenio_search:RecordsSearch"
+
+ORCID_RECORDS_PID_TYPE = 'literature'
+ORCID_RECORDS_DOC_TYPE = 'literature'
+ORCID_RECORDS_PID_FETCHER = 'my_recid_fetcher'
+
 ORCID_WORK_TYPES = {
```

Comment:
This list is valid only in INSPIRE. Should probably be commented out and the default being empty dict.